src/useAsync.ts

```diff
@@ -3,11 +3,11 @@ import useAsyncFn from './useAsyncFn';
 
 export { AsyncState, AsyncFn } from './useAsyncFn';
 
-export default function useAsync<Result = any, Args extends any[] = any[]>(
```

Comment:
maybe `Result` can be with default type like `unknown`? Same as `Promise`'s default

Reply:
Done, it now uses the default `unknown` instead of of `any`. This value will be inferred to something else if the return type of the Promise can be inferred (which is almost always can)